pkg/virt-controller/watch/vmi.go

```diff
@@ -2100,3 +2111,36 @@ func (c *VMIController) getVolumePhaseMessageReason(volume *virtv1.Volume, names
 	}
 	return virtv1.VolumePending, PVCNotReadyReason, "PVC is in phase Lost"
 }
+
+func (c *VMIController) handleDynamicInterfaceRequests(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod) error {
+	log.Log.Object(pod).V(4).Info("network iface hotplug for pod")
+	multusAnnotation, err := services.GenerateMultusCNIAnnotation(vmi)
+	if err != nil {
+		return err
+	}
+
+	currentMultusAnnotation, found := pod.Annotations[networkv1.NetworkAttachmentAnnot]
+	if found {
+		log.Log.Object(pod).V(4).Infof("current multus annotation for pod: %s", currentMultusAnnotation)
+	}
+	if multusAnnotation != "" {
+		pod.Annotations[networkv1.NetworkAttachmentAnnot] = multusAnnotation
+		log.Log.Object(pod).V(4).Infof("updated multus annotation for pod: %s", multusAnnotation)
+	} else {
+		delete(pod.Annotations, networkv1.NetworkAttachmentAnnot)
+		log.Log.Object(pod).V(4).Info("cleared pod cncf.io/networks annotation")
+	}
+
+	if currentMultusAnnotation != multusAnnotation {
+		pod, err = c.clientset.CoreV1().Pods(pod.GetNamespace()).UpdateStatus(context.Background(), pod, v1.UpdateOptions{})
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (c *VMIController) doesPodRequireInterfaceChangeRequests(vmi *virtv1.VirtualMachineInstance) bool {
+	return len(netsetup.NetworksToHotplug(vmi)) > 0
+}
```

Comment:
I think this is redundant and named problematically.
If we try to fix the name, we can get `shouldHotPlugNetworks`, but this is exactly what `len(netsetup.NetworksToHotplug(vmi)) > 0` described directly and there is no need to abstract it.

Reply:
Done.